tools/windows-node-installer/pkg/cloudprovider/azure/vm.go

```diff
@@ -50,6 +50,12 @@ const (
 	vnetRuleName = "vnet_traffic"
 	// winUser is the user used to login into the instance.
 	winUser = "core"
+	// sshPort is the SSH port
+	sshPort = "22"
```

Comment:
If you make these public, won't you be able to re-use them in the test code? or are you not importing this package in the test suite?

Reply:
we haven't imported the package in test suite. Was just following the convention that we followed in the code

Reply:
This is also declared in the [AWS](https://github.com/openshift/windows-machine-config-bootstrapper/blob/master/tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go#L43) pkg. Please have a pre-commit that moves it to a common pkg so that it can be used here.